site/_quarto.yml

```
@@ -1,6 +1,6 @@
 project:
   type: website
-  post-render: make generate-sitemap
+  # post-render: make generate-sitemap
```

Let's undo my random change here:

```suggestion
  post-render: make generate-sitemap
```